designs/gozer/src/ghost.mjs

```diff
@@ -0,0 +1,75 @@
+import { cbqc } from '@freesewing/core'
+
+export const ghost = {
+  name: 'gozer.ghost',
+  measurements: ['hpsToWaistBack', 'waistToFloor', 'head'],
+  draft: ({ measurements, Point, Path, points, paths, Snippet, snippets, sa, macro, part }) => {
+    const eyeSize = measurements.head * 0.0416
+    const size =
+      measurements.hpsToWaistBack + measurements.waistToFloor + measurements.head / Math.PI
+
+    points.middle = new Point(0, 0)
+    points.eyeLine = points.middle.shift(270, measurements.head / Math.PI / 2)
+    points.eyeLeft = points.eyeLine.shift(180, measurements.head * 0.13)
+    points.eyeRight = points.eyeLine.shift(0, measurements.head * 0.13)
+    points.top = new Point(0, -1 * size)
+    points.left = new Point(-1 * size, 0)
+    points.right = new Point(size, 0)
+    points.bottom = new Point(0, size)
+    points.topCp1 = points.top.shift(180, size * cbqc)
+    points.topCp2 = points.top.shift(0, size * cbqc)
+    points.leftCp1 = points.left.shift(270, size * cbqc)
+    points.leftCp2 = points.left.shift(90, size * cbqc)
+    points.bottomCp1 = points.bottom.shift(0, size * cbqc)
+    points.bottomCp2 = points.bottom.shift(180, size * cbqc)
+    points.rightCp1 = points.right.shift(90, size * cbqc)
+    points.rightCp2 = points.right.shift(270, size * cbqc)
+
+    points.eyeLeft.addCircle(eyeSize)
+    points.eyeRight.addCircle(eyeSize)
+
+    paths.seam = new Path()
```

Comment:
You are adding this twice, once as a circle, once as a path.
All you need is:
```js
points.middle = new Point(0, 0).addCircle(size, 'fabric')
```

Reply:
Figured that out now. RTFM